internal/proxy/impl.go

```diff
@@ -1424,33 +1424,38 @@ func (node *Proxy) ShowPartitions(ctx context.Context, request *milvuspb.ShowPar
 	return spt.result, nil
 }
 
-func (node *Proxy) getCollectionProgress(ctx context.Context, request *milvuspb.GetLoadingProgressRequest, collectionID int64) (int64, error) {
+func (node *Proxy) getCollectionProgress(ctx context.Context, msgBase *commonpb.MsgBase, collectionID int64) (int64, error) {
```

Comment:
how about move this function to proxy/util.go?
change msgBase parameter to types.QueryCoord.

Reply:
change function name to getCollectionLoadingProgress